packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -120,10 +126,17 @@ export const ModalContent: React.FunctionComponent<ModalContentProps> = ({
     actions.length > 0 && <ModalBoxFooter>{actions}</ModalBoxFooter>
   );
 
+  const defaultModalBodyAriaRole = bodyAriaLabel ? 'region' : undefined;
+
   const modalBody = hasNoBodyWrapper ? (
     children
   ) : (
-    <ModalBoxBody {...props} {...(!description && !ariaDescribedby && { id: descriptorId })}>
+    <ModalBoxBody
+      aria-label={bodyAriaLabel}
+      role={bodyAriaRole || defaultModalBodyAriaRole}
```

Comment:
So if the consumer defines an `aria-label` but not an `aria-role`, the role defaults to 'region'?

Reply:
Correct the role defaults to 'region' if the consumer passes an aria-role to prevent Axe violations.